components/chosenArticlePage/VotingContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+
+import Button from 'react-bootstrap/Button';
+import patchArticleVotes from '../../utils/patchArticleVotesByArticleId';
+
+
+export default function VotingContainer ({votes, setVotes, chosenArticleId}) {
+
+    const [updateVotes, setUpdateVotes] = useState([])
+    
+    const handleUpVote =  (e) => {
+        setUpdateVotes({
+            inc_votes : 1
+        })
+        }
+        
+        const handleDownVote = (e) => {
+            setUpdateVotes({
+                inc_votes : -1
+            })
+        }
+        
+        
+        useEffect(() => {
+            setVotes(votes)
+        },[votes])
+
+        useEffect(() => {
+             patchArticleVotes({votes, setVotes, chosenArticleId, updateVotes}) 
+        },[updateVotes, setVotes, chosenArticleId])
+
```

Comment:
This component should use optimistic rendering to update the the votes - here are the notes on optimistic rendering to help implement this https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering